source/fundamentals/data-formats/document-data-format-record.txt

```diff
@@ -0,0 +1,141 @@
+.. _fundamentals-records:
+
+=============================
+Document Data Format: Records
+=============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to store and retrieve data in the {+driver-long+}
+using **Java records**. Java records are a type of Java class often used to
```

Comment:
Can we link to some stable Java documentation for records?

Reply:
The link is in the tip below -- is that an acceptable link? Or would you prefer to the one here: https://www.mongodb.com/docs/drivers/java/sync/current/whats-new/#what-s-new-in-4.6

As for placement, while not necessarily consistent across all our documentation, our team prefers to place external links outside the intro paragraph in order to hypothetically maintain the flow of learning from the page.